tests/test_slogpy_console.py

```diff
@@ -0,0 +1,80 @@
+"""The purpose of the code is to give all methods a unit test to make sure that they are working properly"""
+from slogpy.slog import Slog as slog  # noqa: N813
+
+
+def setup_function():
+    """set the Default to no log file, since only interested in console output"""
+    slog.initialize(file_logging=False)
+
+def test_info_console_output(capsys, msg="This is a console test"):
+    """Simple test for console output
+
+    Given:Using slogpy
```

Comment:
Suggestion: Given / When / Then

If you want to write this in a less rigid way, try and use the given / when / then as sentences

Example:
```
Given I am using the slogpy library
When I log an information message
Then the message shows in the stdout stream
And the message does not show in the stderr stream
```